### PR TITLE
Minor style and grammar fixes

### PR DESCRIPTION
**Pull request by pkulikov**

### CODE REVIEW

File: docs/csharp/tutorials/default-interface-members-versions.md [hunk 1]
```diff
@@ -63,7 +63,7 @@ That's a good start. But, the default implementation is too restrictive. Many co
 
 [!code-csharp[VersionTwoImplementation](~/samples/csharp/tutorials/default-interface-members-versions/finished/customer-relationship/ICustomer.cs?name=SnippetLoyaltyDiscountVersionTwo)]
 
-There's many new language capabilities shown in that small code fragment. Interfaces can now include static members, including fields and methods. Different access modifiers are also enabled. The additional fields are private, the new method is public. Any of the modifiers are allowed on interface members.
+There're many new language capabilities shown in that small code fragment. Interfaces can now include static members, including fields and methods. Different access modifiers are also enabled. The additional fields are private, the new method is public. Any of the modifiers are allowed on interface members.
```
Comment (mairaw) [inline docs/csharp/tutorials/default-interface-members-versions.md]:
```suggestion
There are many new language capabilities shown in that small code fragment. Interfaces can now include static members, including fields and methods. Different access modifiers are also enabled. The additional fields are private, the new method is public. Any of the modifiers are allowed on interface members.
```

**Review by mairaw — CHANGES_REQUESTED** [review]
Thank you for your PR @pkulikov. Left you a small suggestion to remove the contraction for the plural. I don't think it's common to contract that expression.

File: docs/csharp/tutorials/default-interface-members-versions.md (hunk 1 above)
Comment (pkulikov) [reply docs/csharp/tutorials/default-interface-members-versions.md]: Thank you @mairaw

**Review by mairaw — APPROVED** [review]
LGTM. Thanks!

**Pull request merged**